src/test/java/seedu/address/logic/listcommands/ListCompaniesCommandTest.java

```diff
@@ -32,9 +30,9 @@ public void execute_listIsNotFiltered_showsSameList() {
         assertCommandSuccess(new ListCompaniesCommand(), model, ListCompaniesCommand.MESSAGE_SUCCESS, expectedModel);
     }
 
-    @Test
-    public void execute_listIsFiltered_showsEverything() {
-        showPersonAtIndex(model, INDEX_FIRST_PERSON);
-        assertCommandSuccess(new ListCompaniesCommand(), model, ListCompaniesCommand.MESSAGE_SUCCESS, expectedModel);
-    }
+    // @Test
+    // public void execute_listIsFiltered_showsEverything() {
+    //     showCompanyAtIndex(model, INDEX_FIRST_COMPANY);
+    //     assertCommandSuccess(new ListCompaniesCommand(), model, ListCompaniesCommand.MESSAGE_SUCCESS, expectedModel);
+    // }
```

Comment:
If the test is not used, perhaps we should delete it?

Reply:
Can replace with a new test I think.

Reply:
Oh, that could be good! What would you suggest?